core/src/main/java/org/eclipse/hono/cache/ExpiringValueCache.java

```diff
@@ -52,6 +52,18 @@
      */
     void put(K key, V value, Duration maxAge);
 
+    /**
+     * Puts a value to the cache but does not change the expiration time of the present value.
```

Comment:
how about calling this method *update* instead?
how about providing some feedback if the operation was successful, e.g. by means of returning a boolean?

Reply:
Naming it _update_ makes sense!

Personally I don't think a return value is required, since it is currently not used. Anyway I added one - it doesn't add to much complexity and maybe it will be useful for someone in future...

Reply:
How about also updating the description of the operation to reflect its new name (update)?